WcaOnRails/spec/controllers/registrations_controller_spec.rb

```diff
@@ -647,30 +647,40 @@
 
     it "sorts 444 by single, and average, and handles ties" do
       registration1 = FactoryBot.create(:registration, :accepted, competition: competition, events: [Event.find("444")])
-      FactoryBot.create :ranks_average, rank: 10, best: 4242, eventId: "444", personId: registration1.personId
-      FactoryBot.create :ranks_single, rank: 20, best: 2000, eventId: "444", personId: registration1.personId
+      FactoryBot.create :ranks_average, rank: 1, best: 2000, eventId: "444", personId: registration1.personId
+      FactoryBot.create :ranks_single, rank: 1, best: 1500, eventId: "444", personId: registration1.personId
 
       registration2 = FactoryBot.create(:registration, :accepted, competition: competition, events: [Event.find("444")])
+      registration2.person.update!(name: "A")
```

Comment:
Why is it important that you have the name set to `A`? As far as I can tell, the names in the resulting psych sheet are not being tested.

Also, if you _really_ need the name to specifically be A,
1. Think whether there is _really_, _really_ a good reason for this
2. If the answer to (1) is Yes, mock a user called `A` using `user_a = FactoryBot.create(:user, :wca_id, name: 'A')` and attach them to the registration like `registration2 = FactoryBot.create(:registration, :accepted, user: user_a, ...etc...)`

Reply:
@gregorbg I did that, because psychsheet is sorted by name (when results are tied), so if I didn't do that, the code might not pass the tests - because the tests check whether the registrations are in the right order, and in the case of the same results, they are sorted by name.